test/arquillian-extension/src/main/java/io/narayana/lra/arquillian/spi/NarayanaLRARecovery.java

```diff
@@ -19,10 +20,25 @@
 
 public class NarayanaLRARecovery implements LRARecoveryService {
     private static final Logger log = Logger.getLogger(NarayanaLRARecovery.class);
+    private static final long WAIT_CALLBACK_TIMEOUT = initWaitForCallbackTimeout();
 
+    /*
+     * Wait for the participant to return the callback. This method does not
+     * guarantee callbacks to finish. The Participant status is not immediately
+     * reflected to the LRA status, but only after a recovery scan which executes an
+     * enlistment. The waiting time can be configurable by
+     * LRAConstants.WAIT_CALLBACK_TIMEOUT property
+     */
     @Override
     public void waitForCallbacks(URI lraId) {
-        // no action needed
+        log.info("Wait for the callback to be returned by the CompletionStage");
+        log.info("waitForCallbacks for: " + lraId.toASCIIString());
+        try {
+            Thread.sleep(WAIT_CALLBACK_TIMEOUT);
+        }
+        catch (InterruptedException e) {
```

Comment:
we need a coding style

Reply:
Narayana and Narayana LRA code style is defined here: https://github.com/jbosstm/narayana-checkstyle-config/tree/main
Any improvements are welcome.